pages/code-review.md

```diff
@@ -1,6 +1,143 @@
+
+
+## Why reviews?
+
+Code reviews are incredibly important step to the development process. Our goals
+when reviewing code are to:
+
+- expose bugs before they make it to production;
+- ensure a consistent quality in the codebase;
+- create an environment for sharing knowledge and developing skills;
+- encourage open communication between the entire team.
+
+
+## Tips for a successful review
+
+For a code review to be successful, we all need to be on the same page. To help
+accomplish this, everyone should follow these tips.
+
+### For everyone
+
+- Remember that we're reviewing the code, not the author. Nothing should be
+  given or taken taken personally.
+- There is often more than one way to approach a solution. Discuss tradeoffs and
+  reach a resolution quickly.
+- Refer to the style guides whenever there's a discussion that can't be resolved.
+- Ask questions rather than make statements. ("What do you think about...?")
+- Ask for clarification if the code or comments are unclear. ("I didn't understand.
+  Can you clarify?")
+- Avoid selective ownership of code. ("mine", "not mine", "yours")
+- Avoid using terms that could be seen as referring to personal traits.
+- Be explicit, people don't always understand your intentions online.
+- Be humble. ("I'm not sure - let's look it up.")
+- Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+- Don't use sarcasm.
+- Keep it real. If emoji, animated gifs, or humor aren't you, don't force them.
+  If they are, use them with aplomb.
+- Talk offline if there is too much back and forth. Post a follow-up comment
+  summarizing the discussion.
+- Praise team members when they create exemplary work or suggestions.
+
+### For code submitters
+
+- Link to the code review from the originating task/issue, if applicable.
+- Remember that the code isn't you, don't get defensive when a reviewer is critical
+  of the code.
+- Seek to understand the reviewer's perspective.
+- Try to respond to every comment.
+- Be grateful for the reviewer's suggestions. ("Good catch, fixing in a4994ec")
+- Explain why the code exists. ("We need to work around these existing patterns")
+- Extract out-of-scope changes and refactorings into future tasks/issues.
+- Push commits based on earlier rounds of feedback as isolated commits to the
+  branch. Do not squash until the branch is ready to merge. Reviewers should be
+  able to read individual updates based on their earlier feedback.
+
+### For code reviewers
+
+- Understand why the code is necessary (bug, user experience, refactoring)
+- Seek to understand the author's perspective.
+- Clearly communicate which ideas you feel strongly about and those you don't.
+- Identify ways to simplify the code while still solving the problem.
+- Offer alternative implementations, but assume the author already considered
+  them. ("What do you think about such-and-such here?")
+- If discussions turn too philosophical or academic, move the discussion to a new
```

Comment:
Indeed - I think the idea is to provide a clear path when there's significant disagreement, in this case favoring the submitter. I'm not sure that we want to sway that direction.

I'll take out the sentence and we can discuss elsewhere?